pkg/osquery/table/platform_tables_linux.go

```diff
@@ -15,7 +15,12 @@ func platformTables(client *osquery.ExtensionManagerClient, logger log.Logger, c
 		gsettings.Settings(client, logger),
 		gsettings.Metadata(client, logger),
 		dataflattentable.TablePluginExec(client, logger,
-			"kolide_nmcli_wifi", dataflattentable.KeyValueType, []string{"/usr/bin/nmcli", "--mode=multiline", "--fields=all", "device", "wifi", "list"},
+			"kolide_nmcli_wifi", dataflattentable.KeyValueType,
+			[]string{"/usr/bin/nmcli", "--mode=multiline", "--fields=all", "device", "wifi", "list"},
 			dataflattentable.WithKVSeparator(":")),
+		dataflattentable.TablePluginExec(client, logger, "kolide_lsblk", dataflattentable.JsonType,
+			[]string{"lsblk", "-J"},
```

Comment:
from the man pages for lsblk: 
> Always explicitly define expected columns by using --output columns-list and --list  in  environments
       where a stable output is required.

We're specifying JSON output, so things probably won't break in a crashy way, but might be good to specify the columns we want just for consistency?

Reply:
Oh hey, docs! Brilliant.

I think `-l` is wrong for us -- it's a different output format. `-P` is interesting, but as long as we have wide support for `-J` that seems simpler. 

I don't think we should be using `--output-columns` either. We don't need consistency, we're parsing EAV style. TBH I wish there was an option to output all the columns, but I don't see it. And it errors on unknowns. So I think that means we either accept the defaults, or plumb through a query style.

I do see a `-a` which would include "empty" devices. 

I can be convinced to iterate on this, but I think I mostly want to get it out and see what we need

Reply:
Getting this out and iterating sounds good to me!